src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -568,7 +555,6 @@ export default class AircraftController {
         // triggering event bus rather than calling locally because multiple classes
         // are listening for the event and aircraft model
         this._eventBus.trigger(EVENT.ADD_AIRCRAFT, aircraftModel);
-        this.initAircraftStripView(aircraftModel);
```

Comment:
I'm very glad to see this go away!

I remember having to keep it around for a reason, though, so I'm curious to see this in action.  This is certainly the correct way to do it; only create a stripView when we need it.